pkg/manager/member/tidb_member_manager.go

```diff
@@ -245,6 +260,105 @@ func (m *tidbMemberManager) syncTiDBStatefulSetForTidbCluster(tc *v1alpha1.TidbC
 	return mngerutils.UpdateStatefulSetWithPrecheck(m.deps, tc, "FailedUpdateTiDBSTS", newTiDBSet, oldTiDBSet)
 }
 
+func (m *tidbMemberManager) syncInitializer(tc *v1alpha1.TidbCluster) error {
```

Comment:
The error will be printed in L121, which means the error logs will be printed twice in this function, what about we print logs in this function directly without printing in L121 and no need to return error from this function.

Reply:
ok